CONTRIBUTING.md

```diff
@@ -1,4 +1,5 @@
 [![MIT Licensed][icon-mit]][license]
+
```

Comment:
Normally badges come in a single line. Let's keep it consistent across kottans repos.